BaseClasses.py

```diff
@@ -1212,13 +1215,13 @@ def add_locations(self, locations: Dict[str, Optional[int]],
             self.locations.append(location_type(self.player, location, address, self))
 
     def add_event(
-        self,
-        location_name: str,
-        item_name: str | None = None,
-        rule: Callable[[CollectionState], bool] | None = None,
-        location_type: type[Location] | None = None,
-        item_type: type[Item] | None = None,
-        show_in_spoiler: bool = True,
+            self,
+            location_name: str,
+            item_name: str | None = None,
+            rule: Callable[[CollectionState], bool] | None = None,
+            location_type: type[Location] | None = None,
+            item_type: type[Item] | None = None,
+            show_in_spoiler: bool = True,
     ) -> Item:
```

Comment:
I think having an extra indent on an argument list isn't preferred when using the style of putting the closing paren/hint on its own line with less indentation.